src/main/java/frc/trigon/robot/motorsimulation/SingleJointedArmSimulation.java

```diff
@@ -0,0 +1,43 @@
+package frc.trigon.robot.motorsimulation;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+
+public class SingleJointedArmSimulation extends MotorSimulation {
+    private final SingleJointedArmSim armSimulation;
+
+    public SingleJointedArmSimulation(DCMotor motor, double gearRatio, double retractedArmLengthMeters, double minimumAngleRadians, double maximumAngleRadians, boolean simulateGravity, double armMass) {
+        armSimulation = new SingleJointedArmSim(motor, gearRatio, SingleJointedArmSim.estimateMOI(retractedArmLengthMeters, armMass), retractedArmLengthMeters, minimumAngleRadians, maximumAngleRadians, simulateGravity, minimumAngleRadians);
+    }
+
+    @Override
+    double calculateFeedforward(double ks, double kg, double kv, double ka, double targetVelocity, double targetPosition) {
+        return ks * Math.signum(Units.rotationsToRadians(targetPosition))
+                + kg * Math.cos(Units.rotationsToRadians(targetPosition)) +
+                kv * Units.rotationsToRadians(targetVelocity) +
+                ka * 0;
```

Comment:
1. You won't always accept these values in rotations
2. You want to compare in the conversion factor
3. The only thing here that must be in radians is the targetPosition that goes inside Math.cos since that accepts radians